src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Helpers.cs

```diff
@@ -72,13 +75,14 @@ private static bool IsAccessible(ISymbol symbol)
             CancellationToken cancellationToken)
         {
             if (searchSymbol == null || symbolToMatch == null)
-            {
                 return false;
-            }
 
             searchSymbol = searchSymbol.GetOriginalUnreducedDefinition();
             symbolToMatch = symbolToMatch.GetOriginalUnreducedDefinition();
 
+            if (searchSymbol.Equals(symbolToMatch))
```

Comment:
Do we need to pass in a symbolcomparer here?

Reply:
no, and i'll comment this.  

1. This is effective a fast bailout to avoid more expensive work below.  so if it returns false, we still do the main full comparison.
2. we actually want this to do equazlity with nullability ignored.  for the purposes of find-refs there's no diff on `string` vs `string?`.